bug: Cannot start-up Cortex on Window with space in path name

### Jan version

0.6.0

### Describe the Bug

This error is related to Windows' handling of UNC (Universal Naming Convention) paths. The issue occurs when CMD.EXE tries to start with a path that begins with \\?\, which is a Windows extended-length path prefix that CMD.EXE doesn't support as a current directory.

This is caused by the naming path in Windows having a space in it causing it to compress like above.

### Steps to Reproduce

1. Install Jan to any drive
2. Open up and click download model
3. Nothing happened, Cortex did not start

### Screenshots / Logs

```
[2025-06-17][15:07:42][app_lib::core::setup][ERROR] [Cortex STDERR]: '\?\C:\Users\ziczac computer\AppData\Local\Programs\Jan'
[2025-06-17][15:07:42][app_lib::core::setup][ERROR] [Cortex STDERR]: CMD.EXE was started with the above path as the current directory.
[2025-06-17][15:07:42][app_lib::core::setup][ERROR] [Cortex STDERR]: UNC paths are not supported.  Defaulting to Windows directory.
```

### What is your OS?

- [ ] MacOS
- [x] Windows
- [ ] Linux

--- a/src-tauri/src/core/setup.rs
+++ b/src-tauri/src/core/setup.rs
@@ -247,7 +247,10 @@ pub fn setup_sidecar(app: &App) -> Result<(), String> {
                 ]);
             #[cfg(target_os = "windows")]
             {
-                cmd = cmd.current_dir(app_handle_for_spawn.path().resource_dir().unwrap());
+                let resource_dir = app_handle_for_spawn.path().resource_dir().unwrap();
+                let normalized_path = resource_dir.to_string_lossy().replace(r"\\?\", "");
+                let normalized_pathbuf = PathBuf::from(normalized_path);
+                cmd = cmd.current_dir(normalized_pathbuf);
             }
 
             #[cfg(not(target_os = "windows"))]

